index.html

```diff
@@ -283,6 +283,29 @@ <h5>Languages</h5>
                     </div>
             </div>
         </section>
-   
+   <section class="footer">
+    <div id="ellipse122"><img src="Ellipse122.png" alt=""></div>
+    <div id="rectangle44"><img src="Rectangle44.png" alt=""></div>
+    <div id="ellipse33"><img src="Ellipse33.png" alt=""></div>
+    <div class="contact"><h6>Contact me</h6></div>
+    <div id="content">
+        <p>
+            If you have an application you are interested in developing, 
+            a feature that you need built or a project that needs coding. 
+            I’d love to help with it 
+        </p>
+    </div>
+    <form action="https://formspree.io/f/xjvlqzdr"  method="post">
+        <input type="text" name="user_name" placeholder="name" id="name" maxlength="30" required>
+        <input type="email" name="user_email" placeholder="E-mail" id="email" required>
+        <textarea name="message" id="message" placeholder="Write your message here" maxlength="500" required ></textarea>
+        <button type="submit"  class="Get-It-Touch"><span>Get It Touch</span></button>
```

Comment:
- [ ] [OPTIONAL] - Great job styling your button as requested by the previous reviewer. It now has something very small left to match the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) you chose. Kindly align it slightly to the right so that it will have a consistent look with your input forms.

**This is how your design currently looks like:**
![image](https://user-images.githubusercontent.com/86318284/178970966-d225c4ef-88c0-4090-b2b4-5faee58359ed.png)

**This is the desired Figma design:**
![image](https://user-images.githubusercontent.com/86318284/178971027-2f0ab6dc-7be4-4065-a61e-fba3d74d4bfd.png)

Reply:
Thank you @Nemwel-Boniface, for approving my projects, I see you suggested a few optional changes I can make, I am going to check on them and improve according to your request.